### PR TITLE
Add bonding interface

### PR DESCRIPTION
**Pull request by HlinaCZ**
Adds doc and test for bond interface

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by phoracek** [comment]
ci add to whitelist

File: docs/user-guide-policy-configure-bond-interface.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+```yaml
+cat <<EOF | ./kubevirtci/cluster-up/kubectl.sh create -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond0-eth1-eth2-policy
+spec:
+  desiredState:
+    interfaces:
+    - name: eth2
```
Comment (SchSeba) [inline docs/user-guide-policy-configure-bond-interface.md]: @EdDev do we need to change the interface ipv4 to false when we make them slaves to the bond or we can remove that?

Comment (phoracek) [reply docs/user-guide-policy-configure-bond-interface.md]: I think we don't need to mention them at all. If there are slaves/ports in bond/bridge, it creates slave connections to respective ifaces. This is redundant.

Comment (phoracek) [reply docs/user-guide-policy-configure-bond-interface.md]: Please see https://github.com/nmstate/kubernetes-nmstate/pull/159

**Comment by phoracek** [comment]
ci test please

**Pull request merged**